tests/core_tests/rct.cpp

```diff
@@ -117,7 +117,7 @@ bool gen_rct_tx_validation_base::generate_with(std::vector<test_event_entry>& ev
     destinations.push_back(td); // 30 -> 7.39 * 4
 
     crypto::secret_key tx_key;
-    bool r = construct_tx_and_get_tx_key(miner_accounts[n].get_keys(), sources, destinations, std::vector<uint8_t>(), rct_txes[n], 0, tx_key, true);
+    bool r = construct_tx_and_get_tx_key(miner_accounts[n].get_keys(), sources, destinations, std::vector<uint8_t>(), false, rct_txes[n], 0, tx_key, true);
```

Comment:
I simply passed `false` to the `is_onetime` parameter of `construct_tx_and_get_tx_key ` in this and other tests for now. Will look into the test code later, and probably will write some tests for this one-time scheme.